conf/modules.config

```diff
@@ -630,7 +637,7 @@ process{
             enabled: !params.no_intervals,
             mode: params.publish_dir_mode,
             path: { "${params.outdir}/variant_calling/${meta.id}/controlfreec" },
-            pattern: "*{mpileup}"
+            pattern: "*mpileup"
```

Comment:
whys is the saveAs thing not here?

Reply:
Since you don't need it when you have a pattern, then the `yml` does not match it and thus is not published